examples/Atlas/+atlasParams/Base.m

```diff
@@ -3,11 +3,12 @@
   % are fixed at compile time for a particular type of plan (walking, manipulating,
   % standing, recovery, etc.) but which can be switched out for a different set when
   % the plan type changes. 
-  properties(GetAccess=public, SetAccess=protected)
+  properties
     body_motion
```

Comment:
can you document here what you mean precisely by 'body_motion' as a param?  (e.g. for every body index listed in the body_motion structure, we can set PID gains for tracking the body position/orientation in the world frame).  correct?